src/CMakeLists.txt

```diff
@@ -101,25 +111,28 @@ set(qpid_dispatch_SOURCES
   router_core/terminus.c
   router_core/transfer.c
   router_core/core_timer.c
+  router_core/module.c
   router_core/modules/edge_router/module.c
   router_core/modules/edge_router/addr_proxy.c
   router_core/modules/edge_router/connection_manager.c
   router_core/modules/edge_router/link_route_proxy.c
   router_core/modules/edge_router/edge_mgmt.c
   router_core/modules/test_hooks/core_test_hooks.c
   router_core/modules/edge_addr_tracking/edge_addr_tracking.c
-  router_core/modules/address_lookup_server/address_lookup_server.c
+  router_core/modules/address_lookup_server/address_lookup_utils.c
```

Comment:
My $0.02:

The address lookup stuff is only used by router-core modules.  It's not used by I/O threads, for example.  This means that we should probably "hide" the functions that are in address_lookup_utils.c by moving that file into router_core/modules/address_lookup_server directory.

Then I would move the address_lookup_server.h file out of the general include/qpid/dispatch directory and put it in src/router_core/ directory.

These changes would result in moving the address lookup stuff into the router_core module, which seems better to me since the address lookup functionality is only to be used by the router_core.

Reply:
Sounds sensible, but it is not directly related to the "create .c file for every .h file" PR I have here. I'll fix the immediate problems and fill a Jira for the additional improvement. https://issues.apache.org/jira/browse/DISPATCH-2285